scripts/secnetperf.ps1

```diff
@@ -204,7 +204,7 @@ $allTests["rps-up-512-down-4000"] = "-exec:lowlat -rstream:1 -up:512 -down:4000
 $hasFailures = $false
 $json["run_args"] = $allTests
 
-function CheckRegressionTput($values, $testid, $transport, $regressionJson) {
+function CheckRegressionResult($values, $testid, $transport, $regressionJson) {
```

Comment:
Can we move this function into the helper file? That's where all helper functions are meant to be.